spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/data/AvroDataTest.java

```diff
@@ -248,17 +269,6 @@ public void testMixedTypes() throws IOException {
     writeAndValidate(schema);
   }
 
-  @Test
-  public void testTimestampWithoutZone() throws IOException {
```

Comment:
Removing this test for `TimestampNTZ` by adding the type to `SUPPORTED_PRIMITIVES` (so that it is handled like any other primitive) is what broke the ORC tests. It looks like the problem is that Spark 3.5's `ColumnarRow` doesn't support `TimestampNTZType`. As a temporary work-around, I've added validation code that checks the value by accessing it as a `TimestampType` instead.

This isn't a change to read behavior, just how we access the data to validate it. I expect to be able to remove this workaround in the next Spark version.

Reply:
yeah I noticed that too and was planning on fixing that in Spark.